Job-based checksum verification failing for `semgrep.yml`

Relates to #4, #41

## Summary

There is an issue in the verification of action checksums on a per-job basis with the `-offline` option in the [`semgrep.yml` workflow](https://github.com/ericcornelissen/ghasum/blob/14091640ac70f35968ef2f6f7e5e2c9e8059b1ad/.github/workflows/semgrep.yml). The [first failing job](https://github.com/ericcornelissen/ghasum/actions/runs/8380291135/job/22949160689#step:5:33) occurred on March 21, 2024 after merging #41. For future reference, the relevant logs from the job are:

```log
[...]
an unexpected error occurred
missing "/home/runner/work/_actions/github/codeql-action/v3.24.7" from cache
exit status 1
Error: Process completed with exit code 1.
```

--- a/.github/workflows/semgrep.yml
+++ b/.github/workflows/semgrep.yml
@@ -27,7 +27,7 @@ jobs:
         WORKFLOW: ${{ github.workflow_ref }}
       run: |
         WORKFLOW=$(echo "$WORKFLOW" | cut -d '@' -f 1 | cut -d '/' -f 3-5)
-        go run ./cmd/ghasum verify -cache /home/runner/work/_actions -offline "$WORKFLOW:$JOB"
+        go run ./cmd/ghasum verify -cache /__w/_actions -offline "$WORKFLOW:$JOB"
     - name: Perform Semgrep analysis
       run: semgrep ci --sarif --output semgrep.sarif
       env:

